buildSrc/build.gradle.kts

```diff
@@ -75,6 +75,10 @@ gradlePlugin {
             id = "transitiveDependencies" // the alias
             implementationClass = "com.datadog.gradle.plugin.transdeps.TransitiveDependenciesPlugin"
         }
+        register("srPayloadUpdate") {
+            id = "srPayloadUpdate" // the alias
+            implementationClass = "com.datadog.gradle.plugin.srpayloadupdate.SrPayloadUpdatePlugin"
+        }
```

Comment:
minor: maybe for the clarity we shouldn't use `sr` and use `sessionReplay` instead in the names (for all classes and tasks). WDYT?

Reply:
yes, makes sense, was afraid they will be too long :)